controllers/fenceagentsremediation_controller_test.go

```diff
@@ -240,21 +244,21 @@ var _ = Describe("FAR Controller", func() {
 				})
 
 				It("should exit immediately without trying to update the status conditions", func() {
-					verifyRemediationTaintExists(workerNode, &farNoExecuteTaint)
+					underTestFAR = verifyPreRemediationSucceed(underTestFAR, workerNode, &farRemediationTaint)
 
 					By("Wait some retries")
 					Eventually(func() int {
-						return plogs.CountOccurences("command failed")
+						return plogs.CountOccurences(cli.FenceAgentFailedCommandMessage)
 					}, "10s", "1s").Should(BeNumerically(">", 3))
 
-					By("Deleting the CR")
-					Expect(k8sClient.Get(context.Background(), client.ObjectKey{Name: workerNode, Namespace: defaultNamespace}, underTestFAR)).To(Succeed())
+					By("Deleting far CR")
```

Comment:
Nit: better use FAR, than "far"

Reply:
I usually address the CR kind by its [short name](https://github.com/medik8s/fence-agents-remediation/blob/main/api/v1alpha1/fenceagentsremediation_types.go#L110) - far.
I might haven't been consistent in the past, but I think it makes sense to follow this logic. At least for new changes.

Reply:
I understand, but "far" is also an English word, using it capitalize should be better, but no strong opinion

Reply:
Good point.  IDK which way is better but for now I will use **FAR**